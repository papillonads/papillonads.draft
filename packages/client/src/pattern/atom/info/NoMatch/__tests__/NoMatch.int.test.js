import { defaultRender } from './NoMatch.int.render'

describe('<NoMatch />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
