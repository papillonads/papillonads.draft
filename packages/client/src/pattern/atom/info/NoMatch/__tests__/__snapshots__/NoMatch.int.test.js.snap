// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <h3>
    No match for 
    <code>
      some-path-name
    </code>
  </h3>
</div>
`;
